public/app/plugins/datasource/azuremonitor/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "types": ["node", "jest", "@testing-library/jest-dom"]
```

Comment:
This isn't necessary for this PR but testing led me to discover types errors in my IDE for all the azuremonitor tests that use testing-library assertions.